addon/components/bs4/bs-button.js

```diff
@@ -1,5 +1,16 @@
 import Button from 'ember-bootstrap/components/base/bs-button';
 
 export default Button.extend({
-  type: 'secondary'
-});
\ No newline at end of file
+  type: 'secondary',
+  classNameBindings: ['outline:btn-outline'],
```

Comment:
This is setting a `btn-outline` class, right? But is this really needed or used somewhere? The examples [here](http://getbootstrap.com/docs/4.1/components/buttons/#outline-buttons) don't mention this class.

Reply:
Oops accidentally left in some test code.